sdk/modelsrepository/Azure.Iot.ModelsRepository/tests/ModelRepoTestBase.cs

```diff
@@ -33,30 +32,18 @@ public static string ParseRootDtmiFromJson(string json)
             return dtmi;
         }
 
-        public static ResolverClient GetTestClient(ClientType clientType, ResolverClientOptions clientOptions = null)
-        {
-            if (clientOptions == null)
-            {
-                clientOptions = new ResolverClientOptions();
-            }
-
-            if (clientType == ClientType.Local)
-            {
-                return new ResolverClient(TestLocalModelRepository, clientOptions);
-            }
-
-            if (clientType == ClientType.Remote)
-            {
-                return new ResolverClient(TestRemoteModelRepository, clientOptions);
-            }
-
-            throw new ArgumentException();
-        }
+        public static readonly string FallbackTestRemoteRepo = ModelRepositoryConstants.DefaultModelsRepository;
 
         public static string TestDirectoryPath => Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
 
         public static string TestLocalModelRepository => Path.Combine(TestDirectoryPath, "TestModelRepo");
 
-        public static string TestRemoteModelRepository => Environment.GetEnvironmentVariable("PNP_TEST_REMOTE_REPO") ?? s_fallbackTestRemoteRepo;
+        public static string TestRemoteModelRepository => Environment.GetEnvironmentVariable("PNP_TEST_REMOTE_REPO") ?? FallbackTestRemoteRepo;
```

Comment:
Since there is no resources to be deployed, I would like to change this value to be straight up the FallbackTestRemoteRepo, unless anyone sees value in having a repo stored in an environment variable.

Reply:
FallbackTestRemoteRepo is actually the default repo so we can even get rid of that.

Reply:
The historical purpose of the env var was to support running remote tests on non prod model repos, for example testing against a forked repo or a test endpoint https://devicemodeltest.azureedge.net/